public/components/trace_analytics/requests/request_handler.ts

```diff
@@ -19,16 +23,41 @@ export function handleDslRequest(http: CoreStart['http'], DSL: any, bodyQuery: a
     if (DSL.query.bool.minimum_should_match)
       bodyQuery.query.bool.minimum_should_match = DSL.query.bool.minimum_should_match;
   }
+  let body = bodyQuery;
+  if (!bodyQuery.index) {
+    body = {...bodyQuery, index: ((mode === 'jaeger' ? JAEGER_INDEX_NAME : DATA_PREPPER_INDEX_NAME)) }
+  }
+  if (mode === 'jaeger' && timeout) {
+    const id = setTimeout(() => setShowTimeoutToast!(), 30000);
+    return http
+      .post(TRACE_ANALYTICS_DSL_ROUTE, {
+        body: JSON.stringify(body),
+      }).then((res) => {
+        return res;
+      })
+      .catch((error) => {
+        console.log(error)
```

Comment:
Can you please change `console.log`to `console.error`?

Reply:
Fixed in [45cdc1e](https://github.com/opensearch-project/dashboards-observability/pull/150/commits/45cdc1e63436848316f6aa1ab5f6f352fcfa9a11)